packages/react-badge/src/components/Badge/index.ts

```diff
@@ -1,5 +1,6 @@
 export * from './Badge';
-export * from './Badge.types';
+// Explicit exports to ommit BadgeCommon
+export type { BadgeSlots, BadgeProps, BadgeState } from './Badge.types';
```

Comment:
nit: Order these alphabetically
```suggestion
export type { BadgeProps, BadgeSlots, BadgeState } from './Badge.types';
```